### PR TITLE
terminalizer, asciinema: update page

### PR DESCRIPTION
**Pull request by vitorhcl**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/terminalizer.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # terminalizer
 
-> Utility program which records the terminal and generate animated GIFs or share a video.
+> Record the terminal, generate animated GIFs and a web player, and optionally upload to terminalizer.com.
```
Comment (kbdharun) [inline pages/common/terminalizer.md:3]:
```suggestion
> Record the terminal, generate animated GIFs and a web player, and optionally upload to <terminalizer.com>.
```

**Review by kbdharun — APPROVED** [review]

File: pages/common/asciinema.md [hunk 2]
```diff
@@ -1,23 +1,24 @@
 # asciinema
 
 > Record and replay terminal sessions, and optionally share them on asciinema.org.
```
Comment (kbdharun) [inline pages/common/asciinema.md:3]:
```suggestion
> Record and replay terminal sessions, and optionally share them on <asciinema.org>.
```
This link should be enclosed within angle brackets too.

File: pages/common/asciinema.md [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
 
-`asciinema rec {{path/to/file.cast}}`
+`asciinema rec {{path/to/recording.cast}}`
 
 - Replay a terminal recording from a local file:
 
-`asciinema play {{path/to/file.cast}}`
+`asciinema play {{path/to/recording.cast}}`
 
 - Replay a terminal recording hosted on asciinema.org:
```
Comment (vitorhcl) [inline pages/common/asciinema.md:23]:
```suggestion
 - Replay a terminal recording hosted on <asciinema.org>:
```

File: pages/common/terminalizer.md [hunk 4]
```diff
@@ -9,16 +10,16 @@
...
 
-`terminalizer play {{path/to/file}}`
+`terminalizer play {{path/to/recording.gif}}`
 
 - Render a recording file as an animated GIF image:
 
-`terminalizer render {{path/to/file}}`
+`terminalizer render {{path/to/recording.gif}}`
 
 - Upload a video to terminalizer.com:
```
Comment (vitorhcl) [inline pages/common/terminalizer.md:23]:
```suggestion
 - Upload a video to <terminalizer.com>:
```

**Review by vitorhcl — COMMENTED** [review]
Should we enclose these links on the example descriptions too?

File: pages/common/asciinema.md [hunk 5]
```diff
@@ -1,23 +1,24 @@
 # asciinema
 
-> Record and replay terminal sessions, and optionally share them on asciinema.org.
+> Record and replay terminal sessions, and optionally share them on <asciinema.org>.
```
Comment (acuteenvy) [inline pages/common/asciinema.md:3]:
```suggestion
> Record and replay terminal sessions, and optionally share them on <https://asciinema.org>.
```

File: pages/common/asciinema.md [hunk 6]
```diff
@@ -1,23 +1,24 @@
...
+> See also: `terminalizer`.
 > More information: <https://asciinema.org/docs/usage>.
```
Comment (acuteenvy) [inline pages/common/asciinema.md:5]: This is a dead link.
```suggestion
> More information: <https://docs.asciinema.org/manual/cli/usage>.
```

File: pages/common/terminalizer.md [hunk 7]
```diff
@@ -1,6 +1,7 @@
 # terminalizer
 
-> Utility program which records the terminal and generate animated GIFs or share a video.
+> Record the terminal, generate animated GIFs and a web player, and optionally upload to <terminalizer.com>.
```
Comment (acuteenvy) [inline pages/common/terminalizer.md:3]:
```suggestion
> Record the terminal, generate animated GIFs and a web player, and optionally upload to <https://terminalizer.com>.
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**